README.Rmd

```diff
@@ -133,7 +134,89 @@ The message printed when the function is ran informs users that the minimum and
 
 ### Elicitation of categorical variables
 
-#### In development
+#### Simulated datasets
+
+Three datasets that simulate the elicitation of categorical variables are included in elicitr:
+
+```{r}
+mechanism_1
+```
+###
+```{r}
+mechanism_2
+```
+###
+```{r}
+mechanism_3
+```
+
+#### Functions
+
+Just like continuous variables, any analysis of categorical variables starts by creating the `elic_cat` object with the function `elic_cat_start()` to store the metadata of the elicitation. To build this `elic_cat` object, four parameters must be specified: 
+
+* `levels` the names of the levels of impact you used in your elicitation
+* `sites` the names of all sites investigated
+* `experts` the maximum number of experts participating in the elicitation process for one mechanism
+* `mechanisms` the names of the mechanisms of impact investigated
+
+```{r}
+my_elicitation_cat <- elic_cat_start(levels = c("level_1", "level_2", "level_3", "level_4", "level_5"),
+                                     sites = c("site_1", "site_2", "site_3", "site_4"),
+                                     experts = 6,
+                                     mechanisms = c("mechanism_1","mechanism_2", "mechanism_3"),
+                                     title = "Categorical elicitation example")
+```
+
+```{r}
+my_elicitation_cat
+```
+
+#### Functions under development
```

Comment:
I would remove all the text in this section. First, there are still many references to the elic_cont variables. Second, we did not write these function yet, so we cannot provide examples.

I would simply list the function under development, e.g.:

* `elic_cat_get_data()`
* `elic_cat_plot()`